packages/react-next/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -468,6 +475,11 @@ export const CalloutContentBase = React.memo(
       return null;
     }
 
+    // We probably just switched focus context to another window. Don't rerender
+    if (!document.hasFocus()) {
```

Comment:
In this case I'm pretty sure you need to use the `getDocument` utility; otherwise this will fail in server-side rendering.

Reply:
This wont get called in server-side rendering, but I'll still make sure we are always getting the right document

Reply:
Ah I missed that this was inside a useEffect, sorry

Reply:
It's not inside a `useEffect`, it's inside the render

Reply:
simply blocked by the check for server-side rendering above it